Makefile

```diff
@@ -102,5 +102,7 @@ test:
 
 .PHONY: check-headers
 check-headers:
-	@./ci/headers-*.sh
+	@./ci/headers-bash.sh; \
+	./ci/headers-docker.sh; \
+	./ci/headers-go.sh; \
```

Comment:
Huh, why did you do it like this with the semicolon and backslash instead of just newlines? Won’t the semicolon cause the exit codes to be ignored?

Reply:
I thinks it's for 
https://github.com/digitalocean/digitalocean-cloud-controller-manager/pull/180#discussion_r259825592?

Reply:
Though, does that mean only the exit code of the last `headers-go.sh` check is going to be respected? 🤔 Wow, I really need to learn more about make...

Reply:
@nanzhong good catch, you're right. I played it safe now and went with @bouk's initial suggestion.